php/8.2/Dockerfile

```diff
@@ -28,7 +28,8 @@ ENV COMPOSER_NO_INTERACTION=1 \
     COMPOSER_MEMORY_LIMIT=-1 \
     TERM=xterm \
     LD_PRELOAD="/usr/lib/preloadable_libiconv.so php" \
-    PHP_CPPFLAGS="$PHP_CPPFLAGS -std=c++11"
+    PHP_CPPFLAGS="$PHP_CPPFLAGS -std=c++11" \
+    PHP_CS_FIXER_IGNORE_ENV=1
```

Comment:
question: should we add a todo to remove this when php-cs-fixer will officially support php 8.2?

Reply:
I think it's always better to be fully safe so we don't leave it for too long.